### PR TITLE
[Service Bus] Add "Message" to batch related names

### LINKED ISSUE #16434: [Service Bus] Use the term "Message" in batch related apis
In the latest v7 of the Service Bus preview package, we have the method `createBatch()` that takes `CreateBatchOptions` and returns a object that has the method `tryAdd()`.

It is being suggested that we add the term "Message" in the above APIs to make it easy for people to find the right method to use

Since we are using the term "message" in all the apis on the sender and receiver, it is not a stretch to do the same for the batch related apis. So, lets do the below renames
- createBatch -> createMessageBatch()
- CreateBatchOptions -> CreateMessageBatchOptions
- tryAdd() -> tryAddMessage()

PRs are welcome to add this feature.
Code Pointers:
- [Set up environment](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#setting-up-your-environment) and run `rush build`
- Open the `sdk/servicebus/service-bus` folder in your editor/IDE and start the renames

cc @JoshLove-msft, @hemanttanwar, @YijunXieMS, @conniey, @srnagar

### PR DESCRIPTION
**Pull request by YijunXieMS**
Do the following renamings:
createBatch -> createMessageBatch()
CreateBatchOptions -> CreateMessageBatchOptions
tryAdd() -> tryAddMessage()

closes #16434

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public void cancelScheduledMessages(Iterable<Long> sequenceNumbers) {
      * @return A {@link ServiceBusMessageBatch} that can fit as many messages as the transport allows.
      */
     public ServiceBusMessageBatch createBatch() {
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java:73]: This method name should also be updated to `createMessageBatch()`.

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java [hunk 2]
```diff
@@ -81,9 +81,9 @@ public ServiceBusMessageBatch createBatch() {
      * @return A new {@link ServiceBusMessageBatch} configured with the given options.
      * @throws NullPointerException if {@code options} is null.
      */
-    public ServiceBusMessageBatch createBatch(CreateBatchOptions options) {
+    public ServiceBusMessageBatch createBatch(CreateMessageBatchOptions options) {
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java:84]: Same here. Update the method name.

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/CreateMessageBatchOptions.java [hunk 3]
```diff
@@ -15,17 +15,17 @@
  * @see ServiceBusSenderAsyncClient
  */
 @Fluent
-public class CreateBatchOptions {
+public class CreateMessageBatchOptions {
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/CreateMessageBatchOptions.java:18]: Make this final.

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageBatch.java [hunk 4]
```diff
@@ -97,7 +97,7 @@ public int getSizeInBytes() {
      * @throws AmqpException if {@code message} is larger than the maximum size of the {@link
      *     ServiceBusMessageBatch}.
      */
-    public boolean tryAdd(final ServiceBusMessage serviceBusMessage) {
+    public boolean tryAddMessage(final ServiceBusMessage serviceBusMessage) {
         if (serviceBusMessage == null) {
             throw logger.logExceptionAsWarning(new IllegalArgumentException("message cannot be null"));
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageBatch.java:102]: This should throw NPE and the error message should be `"'serviceBusMessage' cannot be null"`. Update the JavaDoc too.

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java (hunk 1 above)
Comment (YijunXieMS) [reply sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java]: Updated the sync ones

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java (hunk 2 above)
Comment (YijunXieMS) [reply sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java]: Updated

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/CreateMessageBatchOptions.java (hunk 3 above)
Comment (YijunXieMS) [reply sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/CreateMessageBatchOptions.java]: added

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageBatch.java (hunk 4 above)
Comment (YijunXieMS) [reply sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageBatch.java]: updated

**Review by srnagar — APPROVED** [review]
LGTM!

**Pull request merged**